### DISCUSSION TITLE
user management, etc.

### DISCUSSION THREAD

**Discussion by skynetradio** [agersant/polaris#212] [Ideas]
Hello everyone

I accidentally discovered Polaris on the web and am very excited.
But I have a question regarding user management.
Because I would like to use Polaris to show my listeners what kind of songs my web station has on offer.

For this it would be helpful to only give the user (guest) rights to view the songs but not to play them!!!
(because of copyrights). Is there or will there be an extended user administration at some point.

1.) How to give access only to certain folders
2.) To give access but to block playback
3.) Fix the layout (colors).
4.) Provide information on quality (FLAC,MP3) and bit rate.

In any case, I really like Polaris.
I would be very happy about an answer.

Greetings from Vienna
Thomas

**Comment by aario** [comment]
> 1.) How to give access only to certain folders

I created a pull request: #250
